pkg/api/types.go

```diff
@@ -309,6 +309,7 @@ type DcosConfig struct {
 type MasterProfile struct {
 	Count                    int               `json:"count"`
 	DNSPrefix                string            `json:"dnsPrefix"`
+	MasterSANs               []string          `json:"masterSANs"`
```

Comment:
Let's change this property to `SubjectAltNames`, just to be a little clearer what it is (remove acronym if possible :))